### PR TITLE
Disable wal archiving in the pg_upgrade script instead of Patroni

### PR DESCRIPTION
**Pull request by CyberDem0n**
Preparation for https://github.com/zalando/patroni/pull/1025

### CODE REVIEW

File: postgres-appliance/bootstrap/maybe_pg_upgrade.py [hunk 1]
```diff
@@ -61,6 +65,11 @@ def main():
         raise Exception('Failed to upgrade cluster from {0} to {1}'.format(cluster_version, bin_version))
 
     logger.info('Starting the cluster with new postgres after upgrade')
+    if archive_mode is None:
+        postgresql_config['parameters'].pop('archive_mode')
```
Comment (avaczi) [inline postgres-appliance/bootstrap/maybe_pg_upgrade.py]: didn't you pop this already in line 14?

Comment (CyberDem0n) [reply postgres-appliance/bootstrap/maybe_pg_upgrade.py]: line 17 sets it: https://github.com/zalando/spilo/pull/331/files#diff-70677ed4eab8d8893711a96cb7a8580fR17

**Comment by avaczi** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**